Flashcards.Deviationlark/Database.cs

```diff
@@ -0,0 +1,69 @@
+using System.Configuration;
+using System.Data.SqlClient;
+
+namespace Flashcards
+{
+    class Database
+    {
+        public string connectionString = ConfigurationManager.ConnectionStrings["connString"].ConnectionString;
```

Comment:
Two small points about loading the `connectionString`. You can make it `private readonly` since you only use it in this class and want to prevent updating it in the class. Also, I see you read the `connectionString` from the `ConfigurationManager` in every file that needs it, which is perfectly fine for now, but if you want an extra challenge, try to load the `connectionString` once in `Program.cs` and pass it to the classes that need it. Or pass `ConfigurationManager` to the classes from `Program.cs`. But, don't worry, in the upcoming project Dependency Injection will be covered.